app/assets/javascripts/initializers/initializeReadingListIcons.js

```diff
@@ -33,14 +33,17 @@ function highlightButton(button) {
 }
 
 function addReadingListCountToHomePage() {
-  var user = userData();
-  var readingListCount;
-  if (user && document.getElementById('reading-list-count')) {
-    readingListCount =
-      user.reading_list_ids.length > 0 ? user.reading_list_ids.length : '';
-    document.getElementById('reading-list-count').innerHTML = readingListCount;
-    document.getElementById('reading-list-count').dataset.count =
-      user.reading_list_ids.length;
+  const user = userData();
+  const readingListContainers = document.querySelectorAll(
+    '.reading-list-count',
+  );
```

Comment:
So the weird quirk of our previous implementation was that we only ever populated the first reading list indicator - i.e. the mobile slide-in menu one. This fixes that so we now update both. 

The impact being that users will now see a reading list count on the larger-screen view, when they previously didn't (and probably haven't done for a very long time - I can't recall it ever being there 😆 ). I'm going to tag in @ludwiczakpawel for his thoughts on this, as it could be a bit of a surprise for users and I want to make sure we 100% want this reading list indicator on larger screen.

(I'm mostly bringing it up because I know I would be taken aback to suddenly see this thing tell me I have ~100 things on my reading list on dev.to 😄 )

Reply:
ok @aitchiss @ludwiczakpawel let me know.